tutorials/introductory/customizing.py

```diff
@@ -156,6 +162,9 @@ def plotting_function():
 #    >>> import matplotlib.pyplot as plt
 #    >>> plt.style.use(<style-name>)
 #
+# The approach of using ``mpl_configdir`` should be considered legacy, as it
+# does not allow distribution on PyPI.
```

Comment:
Do we want to go this far?  I think I prefer the method you are advocating here, but the old method is surely not going away at any point is it?

Reply:
It's unlikely to go away for years, but I think it's still good to be opinionated if we think one approach is clearly better (I think mplconfigdir is clearly worse...).

Reply:
I think it depends.  For installing someone else's style file, I agree `pip install cool-style-from-internet` is preferable.   But if I want to write my _own_ style file and stick it somewhere global, `mpl_configdir` is easier than writing my own package and pip-installing it.  I could figure out how to do it, and I'd probably go through the bother because I change machines enough, and its more reproducible because I can make it an environment dependency.  _But_, a lot of our end users aren't going to know how to make their own pip-installable package that contains a style sheet.

Reply:
Sure, let's just remove that for now.